documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -819,27 +819,10 @@ Be aware that fields used for sorting must not be tokenized (see <<field-annotat
 be marked as sortable field using the `@SortableField` annotation (see <<sortablefield-annotation>>).
 ====
 
-
 [WARNING]
 ====
-In the special case of numeric fields indexed through *custom* ``FieldBridge``s, you must provide the SortField type along with the field name.
-This does not apply to numeric fields using default field bridges.
-=====
-[source, JAVA]
------
-QueryBuilder builder = fullTextSession.getSearchFactory()
-    .buildQueryBuilder().forEntity(Book.class).get();
-Query luceneQuery = /* ... */;
-FullTextQuery query = s.createFullTextQuery(luceneQuery, Book.class);
-Sort sort = builder
-  .sort()
-    .byField("authorId", SortField.Type.LONG)//Defining Sort type
-    .andByField("title")
-  .createSort();
-query.setSort(sort);
-List results = query.list();
------
-=====
+If you target fields indexed through *custom* ``FieldBridge``s,
+those field bridges must implement <<metadata-providing-field-bridge,`MetadataProvidingFieldBridge`>>.
```

Comment:
I was trying to say that if, through the Sort DSL, you target fields that are populated by custom field bridges, then those field bridges must implement `MetadataProvidingFieldBridge`.

But I see now that my warning isn't right after the Sort DSL example, like it should be. I pushed an update and rephrased the warning a bit. Is it better?